test/ship_test.rb

```diff
@@ -47,5 +47,8 @@ def test_it_has_sunk
     cruiser.hit
     assert_equal 0, cruiser.health
     assert_equal true, cruiser.sunk?
+    cruiser.hit
+    assert_equal -1, cruiser.health
```

Comment:
I added an additional test to check if everything is valid when the health is a negative integer.

Reply:
I'm curious if the ship's health should be able to go negative, because when a ship is sunk, it shouldn't be able to be hit again. Maybe we can change the hit method logic where a ship can only be hit if it has more than 0 health.